### PR TITLE
test: Add tests for empty validators

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

**Comment by samj1912** [comment]
@jusx as requested - tests for an empty validator

File: __tests__/unit/flex/flex.test.js [hunk 1]
```diff
@@ -94,6 +94,64 @@ describe('Test processBeforeValidate and processAfterValidate invocations', () =
...
+        validate: []
+        pass:
+          - do: comment
+            payload:
+              body: >
+                Thanks for creating a pull request! A maintainer will review your changes shortly. Please don't be discouraged if it takes a while.`
+
+    Helper.mockConfigWithContext(context, config)
+    let registry = { validators: new Map(), actions: new Map() }
+    const commentAction = new Action()
```
Comment (jusx) [inline __tests__/unit/flex/flex.test.js:112]: The change in question where the logic to test the empty array are in each individual action that inherits `Action` -- for that reason, I don't feel this actually is testing the conditions. 

Would be good if the tests are validating the line in question:
`results && results.validationSuites && results.validationSuites.length && results.validationSuites[0].schedule`

Thoughts?

Comment (samj1912) [reply __tests__/unit/flex/flex.test.js]: @jusx fixed :)

**Review by jusx — APPROVED** [review]
Ideally the common logic in question is refactored out to a function or it's super class for better unit testing and coverage but I'm OK with this for now.
